source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Common/EnvironmentSettingNames.cs

```diff
@@ -77,6 +77,10 @@ public static class EnvironmentSettingNames
         public const string ChargeCommandAcceptedSubscriptionName = "CHARGE_COMMAND_ACCEPTED_SUBSCRIPTION_NAME";
         public const string ChargeCommandAcceptedPublishSubscriptionName = "CHARGE_COMMAND_ACCEPTED_PUBLISH_SUBSCRIPTION_NAME";
 
+        // Internal, charge price, confirmed
+        public const string ChargePriceConfirmedSubscriptionName = "CHARGE_PRICE_CONFIRMED_SUBSCRIPTION_NAME";
+        public const string ChargePriceConfirmedDataAvailableSubscriptionName = "CHARGE_PRICE_CONFIRMED_DATAAVAILABLENOTIFIER_SUBSCRIPTION_NAME";
```

Comment:
Perhaps add Command to aligned with other setting names:
```suggestion
        public const string ChargePriceCommandConfirmedSubscriptionName = "CHARGE_PRICE_CONFIRMED_SUBSCRIPTION_NAME";
        public const string ChargePriceCommandConfirmedDataAvailableSubscriptionName = "CHARGE_PRICE_CONFIRMED_DATAAVAILABLENOTIFIER_SUBSCRIPTION_NAME";
```

Reply:
Is it a whole chargeprice command that is confirmed or is it really an operation?

Reply:
The command is only for updating the charge state in DB. When notifying through these domain/internal events we should not use commands any longer